test/integrity_checks.rb

```diff
@@ -332,33 +332,7 @@ def check_parameter_file_format(tsvpath, n_deps, name)
     next if line.start_with? "\#"
 
     # Check endline character
-    if line.include? "\r\n"
-      # Do not perform other checks if the line is the header
-      if i > 1
-        # Check float format
-        # Remove endline character and split the string into array
-        line = line.split("\r\n")[0].split("\t")
-        # For each non dependency entry check format
-        for j in n_deps..line.length() - 1 do
-          # Check for scientific format
-          if (line[j].include?('e-') || line[j].include?('e+') ||
-              line[j].include?('E-') || line[j].include?('E+'))
-            raise "ERROR: Scientific notation found in '#{name}', line '#{i}'."
```

Comment:
Should we remove this check?

Reply:
The `%g` format allows for scientific notation. I actually think this is a good thing as it handles smaller values really well and helps with truncation error.